File: array/max-consecutive-ones.py
class Solution(object):
    def findMaxConsecutiveOnes(self, nums):
        cc=0
        i=0
        maxc=0
        for num in nums:
            if num==1:
                cc+=1
            else:
                cc=0
                
            maxc=max(cc,maxc)
        return maxc

File: array/max-consecutive-ones.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int cc=0,maxct=0;
        for(int num:nums)
        {
            if(num==1)
                cc++;
            else
            {
                cc=0;
            
                }
        maxct=max(maxct,cc);
        }  
        return maxct;
}
};